src/main/scala/org/bitcoins/core/crypto/ECPrivateKey.scala

```diff
@@ -83,11 +83,21 @@ object ECPrivateKey extends Factory[ECPrivateKey] {
     */
   def fromBase58ToPrivateKey(base58 : String) : ECPrivateKey = {
     val decodedBase58 : Seq[Byte] = Base58.decode(base58)
+    //Private keys starting with 'K', 'L', or 'c' correspond to compressed public keys.
+    def isCompressed : Boolean = {
```

Comment:
These two functions should be moved out of the inside of `fromBase58ToPrivateKey`, also `isCompressed` should take in a `Seq[Byte]` as an argument.